.github/ISSUE_TEMPLATE/---question---may-be-a-bug-.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F914 Question?"
+about: Question, proposal, suggestion?
```

Comment:
Isn't proposal covered in "add/request a new feature" issue above?

Reply:
Yeah. Removed `proposal` and `suggestion`